yosys/equiv_simple_clk2fflogic.yosys

```diff
@@ -0,0 +1,10 @@
+hierarchy
```

Comment:
BTW Yosys seems to use `xxx.ys` as the file extension for it's scripts...

Reply:
Done